.eslintrc.js

```diff
@@ -1,295 +1,253 @@
 module.exports = {
-    "env": {},
```

Comment:
I assume this change is only about removing the copyright check

Reply:
Yes, and on save it was formatted...